mono/utils/mono-threads-windows.c

```diff
@@ -86,15 +82,9 @@ mono_threads_suspend_check_suspend_result (MonoThreadInfo *info)
 void
 mono_threads_suspend_abort_syscall (MonoThreadInfo *info)
 {
-	DWORD id = mono_thread_info_get_tid (info);
-	HANDLE handle;
-
-	handle = OpenThread (THREAD_ALL_ACCESS, FALSE, id);
-	g_assert (handle);
-
-	mono_win32_abort_wait (info, handle, id);
-
-	CloseHandle (handle);
+    DWORD id = mono_thread_info_get_tid(info);
+	g_assert (info->native_handle);
```

Comment:
Looks like indentation is a bit screwed up here.

Reply:
Argh, I'll fix this.

Reply:
I've fixed the indentation at a6476ceed1b9c5eef0117f43f6be6789e6deb4b9.